audio/chmap.h

```diff
@@ -22,7 +22,7 @@
 #include <stdbool.h>
 #include "misc/bstr.h"
 
-#define MP_NUM_CHANNELS 16
+#define MP_NUM_CHANNELS 64
```

Comment:
32 would suffice, no?

Reply:
See https://github.com/mpv-player/mpv/pull/11174#issuecomment-1408924972, this also fixes playback of 48ch files. (by playback I mean passing correctly so it can be remixed) And also bitmask for channels is 64-bit so this way we cover all of it.

I had it initially on 32, but hey if there are 48 ch files...

Reply:
Oh huh it's pretty crazy how many channels people use.

Reply:
> Oh huh it's pretty crazy how many channels people use.

Funny that you mention that. I need 128 channels for proper Atmos for Theatres support (without automation for now). Wav ADM 118 channels sample is here. https://dolby.app.box.com/s/bart3gfwblydjysegkltoz08wqlvzspg  I also need that for Sonic for Headphones that also supports 118 dynamic objects and 10 bed.

Can you do that?

Reply:
This is outside of scope of this PR. This file would need first be fixed on ffmpeg side and then here, and custom layouts are not supported at all currently. Well it is something for the future.

One think I noticed is that Windows provide `ISpatialAudioClient` it would be something to implement to properly support those higher channel count, but it is also for another PR.